docs/DeveloperGuide.md

```diff
@@ -625,18 +625,18 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **MSS**
 
 1. User chooses to import the contact list.
-2. KonTActs requests for the file.
-3. User selects the desired file.
-4. KonTActs import the contact list from the specified file and shows successful import.
+2. KonTActs requests for the CSV file.
```

Comment:
I think specifying that it is a CSV file is part of specifying UI details. I think it should be left out or be made more general.